receiver/awsxrayreceiver/README.md

```diff
@@ -0,0 +1,73 @@
+# AWS X-Ray Receiver
+
+**Status: alpha**
+
+## Overview
```

Comment:
Can be separate PR but eventually need to document how to authenticate, just a link to this basically

https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials